README.md

```diff
@@ -140,6 +140,40 @@ previous_searcher = searcher.previous_searcher
 
 ## QuickBooks Online
 
+### Webhooks
+
+Reference: https://developer.intuit.com/app/developer/qbo/docs/develop/webhooks/managing-webhooks-notifications#validating-the-notification
+
+LedgerSync offers an easy way to validate and parse webhook payloads.  It also allows you to easily fetch the resources referenced.  You can create and use a webhook with the following:
+
+```ruby
+# Assuming `request` is the webhook request received from Quickbooks Online
+webhook = LedgerSync::Adaptors::QuickBooksOnline::Webhook.new(
+  payload: request.body.read # It accepts a JSON string or hash
+)
+
+verification_token = WEBHOOK_VERIFICATION_TOKEN # You get this token when you create webhooks in the QuickBooks Online dashboard
+signature = request.headers['intuit-signature']
+raise 'Not valid' unless webhook.valid?(signature: signature, verification_token: verification_token)
+
+# Although not yet used, webhooks may include notifications for multiple realms
+webhook.notifications.each do |notification|
+  puts notification.realm_id
+
+  # Multiple events may be referenced.
+  notification.events.each do |event|
+    puts event.resource # This will perform a Find operation for the resource retrieving the latest version from QuickBooks Online
```

Comment:
Can we expose methods to get the resources without doing finds? In some cases, I feel like we won't have to do that. Like, if an object is deleted. They could be like `raw_resource` and `raw_resources`, and just init the ledger class with the ledger_id

Reply:
I actually was thinking about that.  Will add it in.

Reply:
See if you agree with https://github.com/LedgerSync/ledger_sync/pull/95#discussion_r341659679 which should resolve this.